src/tictactoe/lib/tictactoe/GameService.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Tictactoe.GameService do
+  @callback start_game(String, Map) :: Map
+
+  alias Tictactoe.RoundSetupBuilder, as: RoundSetupBuilder
+  alias Tictactoe.GameRound, as: GameRound
+  alias Tictactoe.GameJsonRepository, as: GameRepository
+
+  def start_game(game_id) do
+    setup = RoundSetupBuilder.init_setup()
+    round = GameRound.start(game_id, setup)
```

Comment:
If you flick the arguments you can use a pipe:

```
saved_id = 
  RoundSetupBuilder.init_setup() // <-- maybe put the id here?
  |> Game.start(game_id)
  |> GameRepository.save()

%{game_id: save_id}
```

Also no need for a "result" temp variable 😄